packages/core/code-loading/webRequire.test.js

```diff
@@ -106,10 +106,11 @@ test('webRequire: should allow using require.extensions like the native node req
 
 test('webRequire: should allow using require.extensions like the native node require (parser)', (t) => {
   const registerStlExtension = (fs, _require) => {
-    const deserializer = require('@jscad/io').stlDeSerializer
+    const { deserializers } = require('@jscad/io')
+    const deserializer = deserializers.stl
```

Comment:
I'm assuming there is little point optimizing imports to (in this case) only get the STL deserializer?  It doesn't reload the code each time I assume?

Reply:
No reload. Just a quick test case for webRequire.